### PR TITLE
Fixed getting badges from inviting youself

### PR DESCRIPTION
**Pull request by not-a-ethan**
Fixed the SQL query for invites based on what [JammyDoger said it should be](https://meta.discourse.org/t/invite-badges-from-inviting-your-self/218143/22?u=not-ethan).

### CODE REVIEW

File: lib/badge_queries.rb [hunk 1]
```diff
@@ -160,11 +160,15 @@ def self.invite_badge(count, trust_level)
         FROM invites i
         JOIN invited_users iu ON iu.invite_id = i.id
         JOIN users u2 ON u2.id = iu.user_id
-        WHERE i.deleted_at IS NULL AND u2.active AND u2.trust_level >= #{trust_level.to_i} AND u2.silenced_till IS NULL
+        WHERE i.deleted_at IS NULL
+        AND i.invited_by_id <> u2.id
+        AND u2.active
+        AND u2.trust_level >= 0
```
Comment (Drenmi) [inline lib/badge_queries.rb:166]: I'm not sure they meant to actually replace the trust level from the method call with 0. 🤔 I believe what they pasted was the query after interpolation with some example values?

Comment (not-a-ethan) [reply lib/badge_queries.rb]: @Drenmi Thanks for pointing that out. I fixed that.

**Review by Drenmi — APPROVED** [review]
LGTM! 🚀 

I noticed we're missing automated tests for all of these. Have you verified the change locally?

**Comment by not-a-ethan** [comment]
@Drenmi What do you mean?

**Comment by Drenmi** [comment]
> What do you mean?

It seems `BadgeQueries` methods are missing tests.

**Comment by not-a-ethan** [comment]
@Drenmi Any updates on getting this merged? Or are you going to make the new checks before we merge this?

**Pull request merged**